specifications/dcp.profiles.md

```diff
@@ -30,3 +40,5 @@ For a usable profile, at least the following aspects must be defined:
 
 In addition, it is possible to further constrain the profile, for example by limiting the number of acceptable
 cryptographic algorithms.
+
```

Comment:
I think we can strike this sentence or word it more precisely. For example, profile authors are recommended, when possible, to use a single credential format.